services/fill.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Start the fill station binary
+After=network.target
+
+[Service]
+Restart=always
+RestartSec=1
+User=root
+WorkingDirectory=/home/crt
+ExecStart=/home/crt/fill_station
```

Comment:
How do we stop this service? Do we have to manually hunt down the pid to kill it?